fivenines_setup.sh

```diff
@@ -29,7 +29,7 @@ echo -n "$1" | sudo tee /etc/fivenines_agent/TOKEN > /dev/null
 
 # Create a system user for the agent
 if ! id -u fivenines >/dev/null 2>&1; then
-  sudo useradd --system --user-group --key USERGROUPS_ENAB=yes fivenines --shell /bin/false --create-home
```

Comment:
We also need to fix the [fivenines_update.sh](https://github.com/Five-Nines-io/fivenines_agent/blob/main/fivenines_update.sh) script to delete the old folder and reinstall in the right place if necessary.

Reply:
for the transition for current users?

ok a moment

Reply:
also patched update script and service file